patroni/utils.py

```diff
@@ -809,7 +809,7 @@ def cluster_as_json(cluster: 'Cluster', global_config: Optional['GlobalConfig']
             role = 'replica'
 
         state = (m.data.get('replication_state', '') if role != 'leader' else '') or m.data.get('state', '')
-        member = {'name': m.name, 'role': role, 'state': state, 'api_url': m.api_url}
+        member = {'scope', global_config.scope, 'name': m.name, 'role': role, 'state': state, 'api_url': m.api_url}
```

Comment:
Oh, you are right. Here the scope isn't available :(
We need to do it here: https://github.com/zalando/patroni/blob/6b7f914da7fd4974ca091c8f55c769b2ad0f58a3/patroni/api.py#L452

Reply:
Not sure what you are mentioning. This function just sends the response.

Reply:
Please suggest the change. It will take significantly less time and efforts.

Reply:
Yes, but we can always modify it before sending:
```python
response = cluster_as_json(cluster, global_config)
response['scope'] = self.server.patroni.postgresql.scope
self._write_json_response(200, response) 
```